samples/Sentry.Samples.Uwp/Sentry.Samples.Uwp.csproj

```diff
@@ -0,0 +1,198 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="15.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
This won't build with `dotnet` so we might have to replace this with some `net5` template of UWP. They are shipping `net5-windows` I believe in 1 or 2 days. Didn't try that in the previews to know if's coming now already